lua/trunk.lua

```diff
@@ -226,6 +257,30 @@ end
 
 -- Startup, including attaching autocmds
 local function start()
+	local cliVersion = getCliVersion()
+	logger.info("Running on CLI version:", cliVersion)
+	if cliVersion == nil then
+		logger.error("nil CLI version")
+		print(
+			"The Trunk Neovim extension requires Trunk CLI version >= 1.17.0 - we could not determine your Trunk CLI version"
+		)
+		print("The extension will not run until you upgrade your CLI version")
```

Comment:
What happens if
a. trunk is not installed (we never tested this as a case from the get-go) or
b. trunk is not init'd in your workspace (`trunk version` should still return latest, but double check plz)

Reply:
I suspect (but will test) that both of these will produce an output that we can't parse as a valid semver, and we'll return nil

Reply:
a. we say that we can't parse your trunk version and don't run
b. if trunk is in path, we report the latest version accordingly, otherwise we say we can't parse the version